sql/core/src/test/scala/org/apache/spark/sql/ConfigBehaviorSuite.scala

```diff
@@ -39,9 +39,8 @@ class ConfigBehaviorSuite extends QueryTest with SharedSparkSession {
     def computeChiSquareTest(): Double = {
       val n = 10000
       // Trigger a sort
-      // Range has range partitioning in its output now. To have a range shuffle, we
-      // need to run a repartition first.
-      val data = spark.range(0, n, 1, 1).repartition(10).sort($"id".desc)
+      // Range has range partitioning in its output now.
```

Comment:
shall we remove this comment now? it's not useful anymore as we do add shuffle, the range output partitioning doesn't matter.

Reply:
okey